frontstage/templates/secure-messages/help/secure-message-select-business-view.html

```diff
@@ -0,0 +1,55 @@
+{% extends "layouts/_block_content.html" %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/textarea/_macro.njk" import onsTextarea %}
+{% from "components/select/_macro.njk" import onsSelect %}
+{% from "components/breadcrumbs/_macro.njk" import onsBreadcrumbs %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+{% block breadcrumbs %}
+    {{
+        onsBreadcrumbs({
+            "ariaLabel": "Breadcrumbs",
+            "id": "breadcrumbs",
+            "itemsList": [{
+                "text": "Back",
+                "url": "/surveys/todo",
```

Comment:
As we could be coming from the todo or the contact us to send a message making this dynamic with `request.referrer` or something similar would improve the flow.

Reply:
Good point. I'll take a look.

Reply:
Updated.

Reply:
This mostly works okay, except as the continue button doesn't work at the moment. If you press continue and try to go back it breaks the chain as the last request was the same page. this should be resolved with the next card but might need looking at again to make sure it functions properly when the full journey is in place.

Reply:
We're going to need to address dynamic navigating on a number of pages if we want to implement it, so this can stay static as /surveys/todo/, i'm not going to block on it.